### DISCUSSION TITLE
How do commands work?

### DISCUSSION THREAD

**Discussion by w9ran** [dawigit/picoclock#5] [Q&A] [answered]
Very cool app!   Just trying to learn how Waveshare RP2040-LCD-1.28 works but could use a little more detail.   After connecting a button to GPIO22 I can change the time and date, and by putting the gyro in the center, change the country.   But I haven't been able to figure out how to access the other commands.  If you could provide a little more detail (words as well as images) I think it would help those who are using this as a learning tool.   Thanks!

**Comment by dawigit** [comment, marked as answer]
Hi w9ran!

To send commands you need to connect your device via usb first.
You need 'minicom' installed.
`sudo apt install minicom`

In a shell:
`minicom -o -D /dev/ttyACM0` 
In a second shell: 
`pigsh 0` 
to start the picoclock-shell (bash script) [and make it connect to /dev/ttyACM**0**]
It should look like this:
`pigsh>`
So: type 'stat' and press return. (Remember your first command will be ignored, so just retry)
After sending the 'stat' command you should see the result of the 'stat' command in the shell you are running 'minicom'.
